src/functions/github/handler.ts

```diff
@@ -0,0 +1,148 @@
+import { Buffer } from "node:buffer";
+import { ellipsis } from "@yuudachi/framework";
+import type { VoiceChannel } from "discord.js";
+import {
+	type AnyThreadChannel,
+	type Message,
+	ThreadAutoArchiveDuration,
+	codeBlock,
+	inlineCode,
+	italic,
+	ChannelType,
+} from "discord.js";
+import kleur from "kleur";
+import { request } from "undici";
+
+// eslint-disable-next-line unicorn/no-unsafe-regex
+const GitHubUrlRegex = /https:\/\/github.com\/(?<org>.+?)\/(?<repo>.+?)\/blob(?<path>\/[^\n#>]+)(?<opts>.+)?/g;
```

Comment:
```suggestion
const GitHubUrlRegex = /https:\/\/github\.com\/(?<org>.+?)\/(?<repo>.+?)\/blob(?<path>\/[^\n#>]+)(?<opts>.+)?/g;
```
Escape the `.` in the regex

Reply:
Thanks, fixed